### DISCUSSION TITLE
Creating tables and columns with postgre always keeps lower case

### DISCUSSION THREAD

**Discussion by lucasNewSharp** [dbeaver/dbeaver#9098] [Q&A] [answered]
#### System information: 
- Windows 10 Version 1909
- DBeaver version: 7.1.0
- Java version: 1.8.0_251

#### Connection specification:
- Database name and version: Postgre 9.5.22
- Driver name: PostgreSQL JDBC Driver

#### Describe the problem you're observing:
On creating tables and columns, even if I use Pascal Case, on save the names are converted to lower case. If I rename the table or the column the name keeps Pascal case.

#### Steps to reproduce, if exist:
Create new table with pascal case naming convention and save, you will see that the table and columns are persistent lower case. If you rename the table or columns the Pascal Case happens.

**Comment by uslss** [comment]
thanks for the bug report

**Comment by serge-rider** [comment, marked as answer]
You should use quotes to set case-sensitive column names.
E.g. "SomeColumnName".

Automatic name conversion was made to avoid typos and unwanted mixed case object creation (which may cause a lot of problems in real applications).
